qutip/solver/parallel.py

```diff
@@ -369,13 +383,11 @@ def loky_pmap(task, values, task_args=None, task_kwargs=None,
         [job.cancel() for job in jobs]
         raise e
 
-    except TimeoutError:
-        [job.cancel() for job in jobs]
-
     finally:
-        executor.shutdown()
+        os.environ['QUTIP_IN_PARALLEL'] = 'FALSE'
```

Comment:
Good catch on moving this into finally. We should have seen that much earlier. Doh.